Common/src/main/java/com/blamejared/crafttweaker/impl/plugin/crafttweaker/BuiltinCraftTweakerPlugin.java

```diff
@@ -0,0 +1,206 @@
+import com.blamejared.crafttweaker.api.bracket.custom.TagBracketHandler;
+import com.blamejared.crafttweaker.api.bracket.custom.TagManagerBracketHandler;
+import com.blamejared.crafttweaker.api.item.IItemStack;
+import com.blamejared.crafttweaker.api.plugin.CraftTweakerPlugin;
+import com.blamejared.crafttweaker.api.plugin.IBracketParserRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.ICommandRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.ICraftTweakerPlugin;
+import com.blamejared.crafttweaker.api.plugin.IJavaNativeIntegrationRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IListenerRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.ILoaderRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IRecipeHandlerRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IScriptLoadSourceRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IScriptRunModuleConfiguratorRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IVillagerTradeRegistrationHandler;
+import com.blamejared.crafttweaker.api.tag.manager.ITagManager;
+import com.blamejared.crafttweaker.api.tag.manager.TagManagerWrapper;
+import com.blamejared.crafttweaker.api.tag.registry.CrTTagRegistryData;
+import com.blamejared.crafttweaker.api.villager.CTTradeObject;
+import com.blamejared.crafttweaker.api.zencode.scriptrun.IScriptRunModuleConfigurator;
+import com.blamejared.crafttweaker.impl.command.type.DumpCommands;
+import com.blamejared.crafttweaker.impl.command.type.HandCommands;
+import com.blamejared.crafttweaker.impl.command.type.InventoryCommands;
+import com.blamejared.crafttweaker.impl.command.type.MiscCommands;
+import com.blamejared.crafttweaker.impl.command.type.ModCommands;
+import com.blamejared.crafttweaker.impl.command.type.RecipeCommands;
+import com.blamejared.crafttweaker.impl.command.type.conflict.ConflictCommand;
+import com.blamejared.crafttweaker.impl.command.type.script.ScriptCommands;
+import com.blamejared.crafttweaker.impl.command.type.script.example.ExamplesCommand;
+import com.blamejared.crafttweaker.mixin.common.access.villager.AccessDyedArmorForEmeralds;
+import com.blamejared.crafttweaker.mixin.common.access.villager.AccessEmeraldForItems;
+import com.blamejared.crafttweaker.mixin.common.access.villager.AccessEnchantedItemForEmeralds;
+import com.blamejared.crafttweaker.mixin.common.access.villager.AccessItemsAndEmeraldsToItems;
+import com.blamejared.crafttweaker.mixin.common.access.villager.AccessItemsForEmeralds;
+import com.blamejared.crafttweaker.mixin.common.access.villager.AccessTippedArrowForItemsAndEmeralds;
+import com.blamejared.crafttweaker.platform.Services;
+import net.minecraft.world.entity.npc.VillagerTrades;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+
+@CraftTweakerPlugin(CraftTweakerConstants.MOD_ID + ":builtin")
+@SuppressWarnings("unused") // Autowired
```

Comment:
The JetBrains annotations for .NET had an `meansImplicitUse` meta-annotation that could be added to annotations.
If something like that also exists in JAva, we could add it to the `CraftTweakerPlugin` annotation so that we no longer need to Suppress the warning here ^^